PendingReleaseNotes

```diff
@@ -63,6 +63,23 @@
     significantly reducing the time of loading inodes for open files. MDS
     creates the table automatically if it does not exist.
 
+  * CephFS snapshot is now stable and enabled by default on new filesystems.
+    To enable snapshot on existing filesystems, use command:
+      - ceph fs set <fs_name> allow_new_snaps
+
+    The on-disk format of snapshot metadata has changed. The old format
+    metadata can not be properly handled in multiple active MDS configuration.
+    To guarantee all snapshot metadata on existing filesystems get updated,
+    perform the following upgrade sequence:
+
+    Reduce the number of ranks to 1 (`ceph fs set <fs_name> max_mds 1`),
+    deactivate all other ranks (`ceph mds deactivate <fs_name>:<n>`),
```

Comment:
Best to refer the admin to the upgrade procedure document for the MDS parts: http://docs.ceph.com/docs/luminous/cephfs/upgrading/#upgrading-the-mds-cluster

Just add a note to skip the step which resets max_mds to its previous setting. Then they can proceed with the scrubbing commands and finally resetting max_mds to the desired setting.